docker/servicesmw/Dockerfile-php

```diff
@@ -0,0 +1,12 @@
+# This is a base Docker image used in prod/preview Jenkinsfile
+FROM artifactory.wikia-inc.com/sus/php-wikia-base:225a68a
```

Comment:
This file looks the same as `Dockerfile-php` from base (except for the `WIKIA_` env passed to `rebuildLocalisationCache` but I don't think they matter when rebuilding messages cache.